docs/source/x_ext.rst

```diff
@@ -427,7 +427,7 @@ A |processor| is allowed to retract its issue request transaction before it is a
 * Set ``issue_valid`` = 0.
 * Keep ``issue_valid`` = 1, but change the ``id`` signal (and if desired change the other signals in ``issue_req``).
 
-The ``instr``, ``mode``, ``id``,  ``ecs``, ``ecs_valid`` and ``rs_valid`` signals are valid when ``issue_valid`` is 1. 
```

Comment:
agree, that makes it more consistent

Reply:
@christian-herber-nxp - you should have `write` permission to merge the PR in

Reply:
I can confirm i do not have the permission: 

> Merging is blocked
> The base branch restricts merging to authorized users. [Learn more about protected branches.](https://docs.github.com/articles/about-protected-branches/)

Maybe some specific settings for this repo?